src/repo/fs/blocks.rs

```diff
@@ -100,6 +100,57 @@ impl FsBlockStore {
             }
         }
     }
+
```

Comment:
Is this move still required? I expected the `Cargo.lock` removal to be the only change.